presto-native-execution/scripts/dockerfiles/README.md

```diff
@@ -38,8 +39,7 @@ cd presto/presto-native-execution && make submodules
 
 ### 2. Build Dependency Image using docker/podman compose
 
-Specify the build type using the ``BUILD_TYPE`` (defaults to Release)
-environment variable.
+The dependencies are always build in Release mode.
```

Comment:
```suggestion
The Dependency Image is always built in ``Release`` mode.
```
Let me know if this suggestion is not correct, please.

Reply:
The Release mode applies to dependencies inside the Dependency image.
But I noticed we mentioned this above as well. So I removed it from here.